libs/application/template-api-modules/src/lib/modules/templates/marriage-conditions-submission/marriage-conditions-submission.service.ts

```diff
@@ -79,9 +79,12 @@ export class MarriageConditionsSubmissionService extends BaseTemplateApiService
     application: { id },
     auth,
   }: TemplateApiModuleActionProps) {
+    const SYSLUMADUR_NATIONAL_ID = '6509142520'
```

Comment:
Just a thought: Seeing this pop up in a couple of places which makes me think this should be stored in a constants file somewhere.